config/sys.config

```diff
@@ -73,6 +73,8 @@
       ]}
   ]},
 
+  {mnesia, [{dir, "data/mnesia"}]},
```

Comment:
I understand this directory is automatically created, right?

When wanting to restart chain with different block version, shall the directory specified here be fresh? (If yes, this shall be hooked in Ansible scripts preferably via user config.)
Current (0.5.0) approach when bumping block version (hard fork) and restarting chain is required includes amending the deployment script so that environments start from fresh DB folder (see [sample commit](https://github.com/aeternity/epoch/commit/6d0837dc287ffd2ec2d583fb4389d6052e2b38b7#diff-2ad1947fe9b869ffd23a6a86872e65e0R14) - via [wiki page on release process](https://github.com/aeternity/epoch/wiki/Release-process)).

Reply:
In the config file, we have `db_path`, but I decided not to reuse that, since it at least initially is likely to point to a place where `aec_persistence` files reside.

One could of course create `DbPath/mnesia`, but relying on anything from epoch.config is tricky until we address the start order problem (since we can't be sure that the config can be read before mnesia starts).

Reply:
So if the directory is deleted, a new schema and empty database will be created in the same location.

Reply:
> In the config file, we have db_path, but I decided not to reuse that, since it at least initially is likely to point to a place where aec_persistence files reside.

FYI In the release notes we explicitly mention whether a persisted blockchain shall not be reused.

Reply:
> One could of course create DbPath/mnesia, but relying on anything from epoch.config is tricky until we address the start order problem (since we can't be sure that the config can be read before mnesia starts).

Does mnesia support symbolic links? If yes, we could hardcode `data/mnesia` (or similar) in the `sys.config`, and then mention in release notes that if user wants DB somewhere else from the default, he/she needs to ensure that `data/mnesia` is a symlink to the desired location.

Reply:
> One could of course create DbPath/mnesia, but relying on anything from epoch.config is tricky until we address the start order problem (since we can't be sure that the config can be read before mnesia starts).

@uwiger My proposal in PR #626, please review.
IMO not getting the DB dir from user config would impact significantly the user experience.